### ISSUE TITLE
Setting of LKT terms when using MF6-generated flow fields

### ISSUE THREAD

**Issue by emorway-usgs** [MODFLOW-USGS/mt3d-usgs#71]
MT3D-USGS needs to access `flowdata` in `BudgetDataModule` in order to set some internal variables that are passed via the linker file  when using MF2K5 (MF-NWT) generated flow fields.

**Comment by emorway-usgs** [comment]
@rbwinst-usgs I was able to get this model to run using the two input files attached to this post.  The two main adjustments were to (1) drop the LKT file type from the name file and (2) and the LAK cells to the SSM input file.  This is a nice test problem and I'd like to add it to our suite of auto-tests run on Travis.  

Note that similar consideration will need to be given to SFR with MODFLOW 6-generated linker input.  That is to say, you won't use SFT with file type FT6 (MODFLOW 6 linker input), you will need to add these boundary cells to SSM and think of them as boundary only, analogous to RIV, for example.  At some point we may revisit support for doing transport in LAK and SFR (and possibly UZF) in MT3D-USGS, but for right now we don't have any readers that can interpret these fluxes from the MODFLOW 6 output.  

Closing this issue, but don't hesitate to let me know (or reopen) if you're still having trouble.  

[LAK_Bug_adjusted_input.zip](https://github.com/MODFLOW-USGS/mt3d-usgs/files/3483078/LAK_Bug_adjusted_input.zip)
